backends/gstreamer/webrtc.rs

```diff
@@ -76,6 +76,7 @@ pub struct GStreamerWebRtcController {
     remote_offer_generation: u32,
     _main_loop: glib::MainLoop,
     data_channels: Arc<Mutex<HashMap<DataChannelId, GStreamerWebRtcDataChannel>>>,
+    data_channels_event_queue: Arc<Mutex<HashMap<DataChannelId, Vec<DataChannelEvent>>>>,
```

Comment:
I get a little bit nervous about races when there are multiple independent locks for what is essentially the same data. Is there any way to do something like:
```rust
enum DataChannelState {
    Buffered(Vec<DataChannelEvent>),
    Created(GStreamerWebRtcDataChannel),
}
```
And then only have a single hashmap?

Reply:
I think we can have a single hashmap adn then just a Sender/Receiver pair, the vec is being used as a queue.

But the enum works too